### PR TITLE
Add ochttp examples

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: examples/grpc/helloworld_client/main.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"os"
 	"time"
 
-	"go.opencensus.io/examples/grpc/exporter"
+	"go.opencensus.io/examples/exporter"
```
Comment (odeke-em) [inline examples/grpc/helloworld_client/main.go]: Wait, does this code compile?

File: examples/grpc/helloworld_server/main.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ import (
 	"net"
 	"net/http"
 
-	"go.opencensus.io/examples/grpc/exporter"
```
Comment (odeke-em) [inline examples/grpc/helloworld_server/main.go]: Ditto, was this perhaps a stray edit?

File: examples/http/helloworld_client/main.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	view.RegisterExporter(exporter)
+	trace.RegisterExporter(exporter)
+
+	view.SetReportingPeriod(1 * time.Second)
+
+	client := &http.Client{
+		Transport: &ochttp.Transport{},
+	}
+
+	for {
```
Comment (odeke-em) [inline examples/http/helloworld_client/main.go]: Please remove this for loop, this is a tight loop, why not just one get
```go
res, err := client.Get(url)
...
```

File: examples/grpc/helloworld_client/main.go (hunk 1 above)
Comment (rakyll) [reply examples/grpc/helloworld_client/main.go]: Yes, I moved the exporter package.

Comment (odeke-em) [reply examples/grpc/helloworld_client/main.go]: Oh I see, sorry for the false alarm
screen shot 2018-02-22 at 4 37 58 pm

File: examples/http/helloworld_client/main.go (hunk 3 above)
Comment (rakyll) [reply examples/http/helloworld_client/main.go]: Done.

**Comment by rakyll** [comment]
PTAL

**Pull request merged**